src/csm_testing/tests/sat_functional/status/test_status.py

```diff
@@ -101,6 +121,42 @@ def test_status_command(self) -> None:
 
         self.assertEqual(expected_header, status_output_header)
 
+    def test_status_formatting_json(self) -> None:
+        """Test that `sat status --format json` returns the properly formatted response"""
+        json_command = 'sat status --format json'
+
+        json_std_out, json_std_err = get_command_output(json_command)
+        json_response = validate_json(json_std_out)
+
+        self.assertTrue(json_response)
+
+        json_header_keys = list(json_response[0].keys())
+
+        status_command = 'sat status'
+        status_output, status_err, status_output_header_str = get_header(status_command, 3)
+        status_all_output = status_output + '\n' + status_err
+        expected_header = adjust_expected_header(SAT_STATUS_HEADER, status_all_output)
```

Comment:
I'm not sure I understand why you have to call `sat status` again to get the expected header lines. Can you explain that? Is it possible to achieve this with just the `json_std_err`? Or do we not omit missing/empty fields when the user specifies JSON or YAML format?